### ISSUE TITLE
Missing contextual Android shortcuts

### ISSUE THREAD

**Issue by oosyrag** [CleverRaven/Cataclysm-DDA#62758]
### Is your feature request related to a problem? Please describe.

Making a list of missing contextual shortcuts off the top of my head and also to add to whenever I run into them. Basically anywhere I need to bring up the soft keyboard to do something, when I otherwise might not need to. Android users please contribute the times you bring up the soft keyboard to do something (that's not typing a word) in the comments below, and I'll add it to the list.

- [ ] While mounted on horse (^)
- [ ] On top of climbable downspout or ladder (>)
- [ ] Inventory dialogue (e)
- [ ] Get item dialogue (e, w, W)
- [ ] Peek where? dialogue (<, >)

### Solution you would like.

Add more contextual shortcuts where suitable.

**Comment by Rimplestilt** [comment]
Yes. Please. As a mainly android user currently I would love to see this. The main issue I have noticed is the lack of certain context shortcuts within some menus.

I had looked into this a while back and was unable to parse exactly how context shortcuts were determined myself.

A major one lacking I found on many menus was "e" for examine, but it extends much further.

**Comment by andrei8l** [comment]
You actually do get all of these shortcuts, but only the very first time you open the window. After that, it depends on which window you opened first. So if you open `i`nventory first, you won't see `TAB` and `F1` in the trade window. If you open the trade window first, you'll see *only* `TAB` and `F1` in other inventory windows. Restarting the game doesn't revert the shortcuts either.

So the problem is that the shortcuts are not properly saved and/or cleared.
